src/PIL/PngImagePlugin.py

```diff
@@ -76,21 +76,50 @@
 
 _simple_palette = re.compile(b"^\xff*\x00\xff*$")
 
-# Maximum decompressed size for a iTXt or zTXt chunk.
-# Eliminates decompression bombs where compressed chunks can expand 1000x
 MAX_TEXT_CHUNK = ImageFile.SAFEBLOCK
-# Set the maximum total text chunk size.
+"""
+Maximum decompressed size for a iTXt or zTXt chunk.
+Eliminates decompression bombs where compressed chunks can expand 1000x
```

Comment:
```suggestion
Eliminates decompression bombs where compressed chunks can expand 1000x.
```

Add a fullstop:

![image](https://user-images.githubusercontent.com/1324225/87130500-aad49780-c29b-11ea-9702-3e3a0a8bf052.png)